static/app/views/dashboardsV2/data.tsx

```diff
@@ -230,87 +426,210 @@ export const DASHBOARDS_TEMPLATES: DashboardDetails[] = [
+              'transaction',
+              'count_if(transaction.status,equals,internal_error)',
+              'count()',
+            ],
+            conditions: 'count():>100',
+            orderby: '-equation[0]',
           },
         ],
+        widgetType: WidgetType.DISCOVER,
         tempId: uniqueId(),
       },
       {
-        title: t('Apdex over time'),
-        displayType: DisplayType.LINE,
+        title: t('Top 5 Issues by Unique Users'),
+        displayType: DisplayType.TABLE,
         interval: '5m',
         queries: [
           {
-            name: t('Apdex'),
-            conditions: 'event.type:transaction',
-            fields: ['apdex()'],
-            orderby: 'apdex',
+            name: '',
+            fields: ['issue', 'count_unique(user)', 'title'],
+            conditions: '',
+            orderby: '-count_unique_user',
           },
         ],
+        widgetType: WidgetType.DISCOVER,
         tempId: uniqueId(),
       },
       {
-        title: t('Failure Transactions over time'),
+        title: t('Transactions Erroring'),
+        displayType: DisplayType.TABLE,
+        interval: '5m',
+        queries: [
+          {
+            name: '',
+            fields: ['count()', 'transaction'],
+            conditions: 'transaction.status:internal_error',
+            orderby: '-count',
+          },
+        ],
+        widgetType: WidgetType.DISCOVER,
+        tempId: uniqueId(),
+      },
+      {
+        title: t('Issues assigned to Me or My Team'),
+        displayType: DisplayType.TABLE,
+        interval: '5m',
+        queries: [
+          {
+            name: '',
+            fields: ['assignee', 'issue', 'title'],
+            conditions: 'assigned:me',
+            orderby: 'priority',
+          },
+        ],
+        widgetType: WidgetType.ISSUE,
+        tempId: uniqueId(),
+      },
+      {
+        title: t('p75(duration)'),
+        displayType: DisplayType.WORLD_MAP,
+        interval: '5m',
+        queries: [
+          {
+            name: '',
+            fields: ['p75(transaction.duration)'],
+            conditions: '',
+            orderby: '',
+          },
+        ],
+        widgetType: WidgetType.DISCOVER,
+        tempId: uniqueId(),
+      },
+      {
+        title: t('p75 Over Time'),
         displayType: DisplayType.LINE,
         interval: '5m',
         queries: [
           {
-            name: t('Failing Transactions'),
-            conditions: 'event.type:transaction',
-            fields: ['failure_count()'],
-            orderby: 'failure_count',
+            name: '',
+            fields: ['p75(transaction.duration)'],
+            conditions: '',
+            orderby: '',
           },
         ],
+        widgetType: WidgetType.DISCOVER,
         tempId: uniqueId(),
       },
       {
-        title: t('Misery over time'),
+        title: t('Throughput (Transactions Per Minute)'),
```

Comment:
Question, does Events Per Minute make more sense here since it encompasses Transaction events and Error events?